Source/WebCore/platform/audio/glib/MediaSessionGLib.cpp

```diff
@@ -284,9 +284,13 @@ void MediaSessionGLib::updateNowPlaying(NowPlayingInfo& nowPlayingInfo)
 
     GVariantBuilder propertiesBuilder;
     g_variant_builder_init(&propertiesBuilder, G_VARIANT_TYPE("a{sv}"));
-    g_variant_builder_add(&propertiesBuilder, "{sv}", "Metadata", getMetadataAsGVariant(nowPlayingInfo));
-    emitPropertiesChanged(g_variant_new("(sa{sv}as)", DBUS_MPRIS_PLAYER_INTERFACE, &propertiesBuilder, nullptr));
+
+    auto metadata = getMetadataAsGVariant(nowPlayingInfo);
+    g_variant_builder_add(&propertiesBuilder, "{sv}", "Metadata", metadata);
```

Comment:
`getMetadataAsGVariant` returns a floating `GVariant*` so it should be safe to pass it directly to `g_variant_builder_add()`. No need to split it onto a separate line like this.